app/src/main/java/umc/cozymate/ui/my_page/favorite/MyFavoriteActivity.kt

```
@@ -60,11 +64,12 @@ class MyFavoriteActivity : AppCompatActivity() {
         membersAdapter = FavoriteRoommateRVAdapter(emptyList()) { memberId ->
-            val intent = Intent(this, CozyHomeRoommateDetailActivity::class.java).apply {
-                putExtra("member_id", memberId) // 멤버 아이디 전달
-            }
+//            val intent = Intent(this, CozyHomeRoommateDetailActivity::class.java).apply {
+//                putExtra("member_id", memberId) // 멤버 아이디 전달
+//            }
             try {
-                startActivity(intent)
+                detailViewModel.getOtherUserDetailInfo(memberId)
+            //startActivity(intent)
             } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)